src/playback/qml/MuseScore/Playback/internal/AuxSendControl.qml

```diff
@@ -81,7 +81,8 @@ Item {
             readonly property bool isHovering: bypassBtn.mouseArea.containsMouse
 
             Layout.fillWidth: true
-            Layout.fillHeight: true
+            Layout.preferredHeight: audioSignalAmountKnob.backgroundHeight
```

Comment:
Could you explain why you need this? This button by design should use 24 (that's why Layout.fillHeight was here)